listenbrainz/webserver/static/js/src/RecentListens.test.tsx

```diff
@@ -1,10 +1,17 @@
+/* eslint-disable dot-notation */
```

Comment:
Can we do this on the specific lines where it causes problems, doing eslint-disables on entire files is something i'd like to avoid.